spec/system/task_spec.rb

```diff
@@ -100,13 +92,12 @@
   describe 'Task削除' do
     context '正常系' do
       # FIXME: テストが失敗するので修正してください
-      xit 'Taskが削除されること' do
-        project = FactoryBot.create(:project)
-        task = FactoryBot.create(:task, project_id: project.id)
+      fit 'Taskが削除されること' do
```

Comment:
focus設定がついたままのコミットとなっています。
```suggestion
      it 'Taskが削除されること' do
```

Reply:
b761505 で修正しました。